spot-client/src/spot-remote/ui/components/remote-control-menu/volume-button.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { ExpandLess, ExpandMore } from 'common/icons';
+import { remoteControlService } from 'common/remote-control';
+
+const TYPE_DOWN = 'down';
+const TYPE_UP = 'up';
+
+/**
+ * Implements a volume control button.
+ */
+export default class VolumeButton extends React.Component {
+    static propTypes = {
+        type: PropTypes.oneOf(TYPE_DOWN, TYPE_UP)
```

Comment:
I think there's a propType warning from this. `A common mistake is to write oneOf(x, y, z) instead of oneOf([x, y, z]).`